packages/events/ReactGenericBatching.js

```diff
@@ -46,26 +47,31 @@ export function batchedUpdates(fn, bookkeeping) {
       // If a controlled event was fired, we may need to restore the state of
       // the DOM node back to the controlled value. This is necessary when React
       // bails out of the update without touching the DOM.
-      _flushInteractiveUpdatesImpl();
+      flushDiscreteUpdatesImpl();
       restoreStateIfNeeded();
     }
   }
 }
 
-export function interactiveUpdates(fn, a, b, c) {
-  return _interactiveUpdatesImpl(fn, a, b, c);
+export function discreteUpdates(fn, a, b, c) {
+  return discreteUpdatesImpl(fn, a, b, c);
 }
 
-export function flushInteractiveUpdates() {
-  return _flushInteractiveUpdatesImpl();
+let lastInteractiveTimeStamp = 0;
```

Comment:
Hmm is this the right place to put this? (cc: @sebmarkbage) Seems good enough to merge but it wouldn't have been my first guess.

Reply:
Seemed like the most sane place to put it, but I'm not bothered, it just meant less indirection on the existing codebase.

Reply:
Addressed this in latest commit.